.gitlab/maintenance_jobs/kitchen.yml

```diff
@@ -50,12 +50,7 @@ periodic_kitchen_cleanup_ec2:
   rules:
     !reference [.on_main_or_testing_cleanup]
   script:
-    - export FOUR_HOURS_AGO=`date --date="-4 hours" "+%Y-%m-%d %H:%M:%S"`
-    - export ZOMBIES=`aws ec2 describe-instances --filters "Name=tag:CreatedBy,Values=datadog-agent-kitchen-tests" --region us-east-1 --query "Reservations[*].Instances[?LaunchTime<='$FOUR_HOURS_AGO'].[InstanceId]" --output text`
-    - echo $ZOMBIES
     # Only run aws ec2 terminate-instances if $ZOMBIES is not empty, otherwise the command fails
     # https://docs.aws.amazon.com/cli/latest/reference/ec2/terminate-instances.html#options
-    - |
-      if [ -n "$ZOMBIES" ]; then
-        echo "$ZOMBIES" | xargs -I% aws ec2 terminate-instances --region us-east-1 --instance-ids %
-      fi
+    - export AWS_DEFAULT_REGION=us-east-1
+    - python3.6 /deploy_scripts/cleanup_ec2.py
```

Comment:
could (should) it be a more recent version of python? Or it's the one installed on the image?

Reply:
We use `python3.6` because the image on top of which our image is built is `ubuntu:18.04`. Not sure it is worth updating it unless we really have a use case for this